packages/asc-ui/src/components/GlobalStyle/GlobalStyle.ts

```diff
@@ -8,8 +8,6 @@ const GlobalStyle = createGlobalStyle`
 
   body {
     font-family: ${getValueFromTheme('typography.fontFamily')};
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
```

Comment:
I think this was from a merge conflict that was perhaps resolved incorrectly? I don't think we want to remove this.